File: spring-cloud-integrated-demo/discovery-server/src/main/scala/com/github/jccode/springcloud/integrateddemo/discoveryserver/Application.scala
package com.github.jccode.springcloud.integrateddemo.discoveryserver

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.cloud.netflix.eureka.server.EnableEurekaServer

@EnableEurekaServer
@SpringBootApplication
class Application {

}

object Application {
  def main(args: Array[String]): Unit = {
    SpringApplication.run(classOf[Application], args: _*)
  }
}

File: spring-boot-security/zuul-demo/api-gateway/src/main/java/com/github/jccode/springsecuritydemo/zuuldemo/apigateway/Application.java
package com.github.jccode.springsecuritydemo.zuuldemo.apigateway;

import com.github.jccode.springsecuritydemo.zuuldemo.apigateway.filter.route.OAuth2HeaderFilter;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.netflix.zuul.EnableZuulProxy;
import org.springframework.context.annotation.Bean;

/**
 * Application
 *
 * @author 01372461
 */
@EnableZuulProxy
@SpringBootApplication
public class Application {

    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }

    @Bean
    public OAuth2HeaderFilter oauth2HeaderFilter() {
        return new OAuth2HeaderFilter();
    }
}
